pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitDisplayNameTest.java

```diff
@@ -0,0 +1,102 @@
+        subkitSettingsIds = KitUtil.createSubKitsForTheStudy("BLOOD",  "BLOOD and RNA", 0, "RNA", null, 1);
+        Assert.assertEquals(2, subkitSettingsIds.length);
+        NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService();
+        DDPInstance ddpInstance = DDPInstance.getDDPInstanceWithRole("test_instance",DBConstants.JUNIPER_STUDY_INSTANCE_ROLE);
+        JuniperKitRequest juniperTestKit = generateKitRequestJson();
+        KitUtil.createNonPepperTestKit(juniperTestKit, nonPepperKitCreationService, ddpInstance);
+        List<KitRequestShipping> kits = KitRequestShipping.getKitRequestsByRealm("test_instance", "overview", BLOOD_RNA_KIT_TYPE_NAME);
+        Assert.assertEquals(1, kits.size());
+        Assert.assertEquals("BLOOD", kits.get(0).getKitTypeName());
+        Assert.assertEquals("BLOOD and RNA", kits.get(0).getDisplayName());
+
+    }
+
+    @Test
+    public void testEmptyDisplayName() {
+        KitRequestShipping kitWithDisplayName = new KitRequestShipping(guid, "TestProject_2", null, "FAKE_DSM_LABEL_UID", "study", BLOOD_RNA_KIT_TYPE_NAME, 1L, 1L,
+                "https://easypost-files.s3-us-west-2.amazonaws"
+                        + ".com/files/postage_label/20200214/8240f1b66535494a82b1ec0d566c3f0f.png", "", "794685038506",
+                "9405536897846100551129", "https://track.easypost.com/djE6dHJrXzY4NGJmYzU3ZjM5OTQ1Zjg5MjEzOGRmMWVmMjI1NWZl",
+                null, 12L, false, "", 12L, null, 12L, "so what", "mf_testLabel", false, "shp_f470591c3fb441a68dbb9b76ecf3bb3d",
+                12L, null, "44445", false, "NOT FOUND", null, null, null, null, 0L, false, "STANDALONE", null, null, null,
+                null, null, null, null, null, null, BLOOD_RNA_KIT_TYPE_DISPLAY_NAME);
+        Assert.assertEquals(kitWithDisplayName.getDisplayName(), BLOOD_RNA_KIT_TYPE_DISPLAY_NAME);
+        Assert.assertEquals(kitWithDisplayName.getKitTypeName(), BLOOD_RNA_KIT_TYPE_NAME);
+
+        // test that not having a display name means the display name is set as the kit type name
+        KitRequestShipping kitWithoutDisplayName = new KitRequestShipping(guid, "TestProject_2", null, "FAKE_DSM_LABEL_UID", "study", BLOOD_RNA_KIT_TYPE_NAME, 1L, 1L,
+                "https://easypost-files.s3-us-west-2.amazonaws"
+                        + ".com/files/postage_label/20200214/8240f1b66535494a82b1ec0d566c3f0f.png", "", "794685038506",
+                "9405536897846100551129", "https://track.easypost.com/djE6dHJrXzY4NGJmYzU3ZjM5OTQ1Zjg5MjEzOGRmMWVmMjI1NWZl",
+                null, 12L, false, "", 12L, null, 12L, "so what", "mf_testLabel", false, "shp_f470591c3fb441a68dbb9b76ecf3bb3d",
+                12L, null, "44445", false, "NOT FOUND", null, null, null, null, 0L, false, "STANDALONE", null, null, null,
+                null, null, null, null, null, null, null);
+        Assert.assertEquals(kitWithoutDisplayName.getDisplayName(), BLOOD_RNA_KIT_TYPE_NAME);
```

Comment:
I think the args here are reversed.  `BLOD_RNA_KIT_TYPE_NAME` is the expected value and should be the first param, right?